assembly/src/etc/tornado.properties

```diff
@@ -1,8 +1,35 @@
 tornado.opencl.print = false
 tornado.debug = false
-tornado.platform = 0
+#
+# Index of the default driver (OpenCL vs PTX) used for TaskSchedule.
```

Comment:
Update the description to this:

Index of the default TornadoVM's driver (either OpenCL or PTX backend).
Please, check the output of the `tornado --devices` command for the desired value.